app.py

```diff
@@ -9,6 +10,26 @@
 from urllib3.exceptions import SSLError
 from collections import OrderedDict
 from typing import Callable
+from fast_autocomplete import AutoComplete
+
+AUTOCOMPLETE_MAX_COST = 3
+AUTOCOMPLETE_SIZE = 5
+
+#codes inspired -> JKL project owner TOM ..
+
+words = {}
```

Comment:
**[Required]** Can remove this, not needed

Reply:
ok